src/lib/core/overlay/overlay.spec.ts

```diff
@@ -336,6 +336,24 @@ describe('Overlay', () => {
 
   });
 
+  describe('panelClass', () => {
+    let config: OverlayState;
+
+    beforeEach(() => {
+      config = new OverlayState();
+      config.panelClass = 'custom-panel-class';
+    });
```

Comment:
You don't really need the `beforeEach` here; you can create the stats and set the class in the test itself because it's only being used once